.gitignore

```diff
@@ -50,6 +50,7 @@ examples/*
 !examples/*.py
 !examples/run_examples.sh
 _tmp_*
+.dmypy.json
```

Comment:
dmypy is a service that runs mypy as a demon, making mypy checks nearly instant.